src/components/HomePage/ProductPlacementArea/ProductDescription.js

```diff
@@ -13,28 +20,56 @@ import useSession from "../../../hooks/useSession";
 
 export const ProductDescription = React.forwardRef(({ toggleLoginModal }, ref) => {
     const { isLoggedIn } = useSession();
+
+    const classes = useProductDescriptionStyles({ isMobile: !useDesktop() });
+
     return (
         <div
             ref={ref}
-            style={{ height: "500px" }}
+            className={classes.productDescription}
         >
-            <Typography>
-                Other content, like what is this for and cool vector and stock images with no copyright
-            </Typography>
-
-            {!isLoggedIn &&
-            <>
-                Are you a company?
-                <Button
-                    variant="text"
-                    color="primary"
-                    onClick={toggleLoginModal}
-                >
-                Login
-                </Button>
-                <RouterLink to="/apply/company">Join Us</RouterLink>
-            </>
-            }
+            <div className={classes.productDescriptionCol}>
+                <StudentSvg />
+
+                <div className={classes.productDescriptionInfo}>
+                    <Typography>
+                        As a Student, you can search for job offers according to your interests.
+                    </Typography>
+                </div>
+
+            </div>
+            <div className={classes.productDescriptionCol}>
+                <CompanySvg />
+
+                <div className={classes.productDescriptionInfo}>
+                    <Typography>
+                        As a Company, you can advertise yout job opportunities to students!
+                    </Typography>
+
+                    {!isLoggedIn &&
+                    <>
+                        <Typography>
+                            Are you a company?
+                        </Typography>
+                        <Button
+                            variant="text"
+                            color="primary"
+                            onClick={toggleLoginModal}
+                        >
+                        Login
+                        </Button>
+                        <Button
+                            variant="text"
+                            color="primary"
+                        >
+                            <RouterLink className={classes.productDescriptionAnchor} to="/apply/company">Join Us</RouterLink>
+                        </Button>
```

Comment:
You can use the `component` prop to change the displayed component, instead of using children. https://material-ui.com/guides/composition/#button

That might be helpful for avoiding manual styling as well.

```suggestion
                        <Button
                            color="primary"
                            component={RouterLink}
                            to="/apply/company"
                        >
                            Join Us
                        </Button>
```

Reply:
When making these changes, the page sends this warnings:
![image](https://user-images.githubusercontent.com/49153615/114019158-f8e7ed00-9865-11eb-8328-67dc2f082dd7.png)
![image](https://user-images.githubusercontent.com/49153615/114019218-08673600-9866-11eb-80b7-8c0cba3d9474.png)
![image](https://user-images.githubusercontent.com/49153615/114019259-161cbb80-9866-11eb-9720-b2bec2877011.png)

Reply:
As discussed "offline", you need to make RouterLink a "refForwarder". Coincidentally, that pattern is already done inside RouterLink, but not on the required level. You need to copy that "forwardRef" call to the RouterLink function itself (not inside - and do not remove the existing forwardRef call!) and pass the ref to the Link component